lib/semian.rb

```diff
@@ -153,7 +153,11 @@ def register(name, **options)
     circuit_breaker = create_circuit_breaker(name, **options)
     bulkhead = create_bulkhead(name, **options)
 
-    resources[name] = ProtectedResource.new(bulkhead, circuit_breaker)
+    if circuit_breaker.nil? && bulkhead.nil?
+      raise ArgumentError, 'Both bulkhead and circuitbreaker configured off.'
```

Comment:
Disabled might be more natural here in the error "configured off" sounds a little strange.